src/utils.js

```diff
@@ -200,8 +247,7 @@ function stackString (e) {
 function emptyNode (node) {
   const nodes = node.childNodes
   const len = nodes.length
-  let i
-  for (i = len - 1; i >= 0; i--) node.removeChild(nodes[i])
+  for (let i = len - 1; i >= 0; i--) node.removeChild(nodes[i])
```

Comment:
Again, please use `nodes.forEach()` instead of for loops - You don't really need to refer to the items via their index, after all.